cedar-policy-core/src/ast/policy_set.rs

```diff
@@ -264,6 +264,125 @@ impl PolicySet {
         Ok(())
     }
 
+    /// Helper function for merge_policyset to check if the `PolicyID` pid
+    /// appears in this `PolicySet`'s links or templates.
+    fn policy_id_is_bound(&self, pid: &PolicyID) -> bool {
+        match self.templates.get(pid) {
+            Some(_) => true,
+            None => match self.links.get(pid) {
+                Some(_) => true,
+                None => false,
```

Comment:
If you don't care about the contents you can use `is_some()`/`is_none()` to create a simpler expression.